node/src/reactor/initializer.rs

```diff
@@ -59,11 +60,15 @@ pub(crate) enum Event {
     /// Storage request.
     #[from]
     StorageRequest(StorageRequest),
+
+    /// Contract runtime request.
+    #[from]
+    ContractRuntimeRequest(Box<ContractRuntimeRequest>),
```

Comment:
Why do we need this variant if a `ContractRuntimeRequest` can also be a `contract_runtime::Event`?

Reply:
This has been changed in https://github.com/casper-network/casper-node/pull/2614/commits/319ff105afcd205c0e142956c778a8742a5a6d1c

Reply:
As discussed, the change is reverted for now in d7f147fe8b0162bb44534ad7554ce01a37caa323 as this will be a part of bigger refactoring.